luafilesystem-1.7.0-2.rockspec

```diff
@@ -0,0 +1,28 @@
+package = "LuaFileSystem"
```

Comment:
It looks really counter-intuitive to have blocking FS utilites in our official rocks repository when we have the built-in fio module. It is better to rewrite the LuaFileSystem API on top of fio for compatibility with third-party modules.

Reply:
See summary below.